README.md

```diff
@@ -21,6 +21,49 @@ framework
 * [ReportPortalPortal](https://github.com/znsio/ReportPortalIntegration/blob/main/README.md)
 * [ApplitoolsIntegration](https://github.com/znsio/ApplitoolsIntegration/blob/main/README.md)
```

Comment:
Update link: https://github.com/znsio/ApplitoolsIntegration/blob/main/README.md#how-to-configure-applitools-visual-testing-in-your-automation-framework

Reply:
Updated the link